telos-frontend/src/components/todo/JournalTodo.jsx

```diff
@@ -45,30 +47,30 @@ const useStyles = makeStyles({
   },
 });
 
-const JournalTodo = () => {
-  const listitems = [
-    {
-      name: 'OnGoing',
-      onGoing: true,
-      completed: false,
-    },
-    {
-      name: 'OutDated',
-      onGoing: false,
-      completed: true,
-    },
-  ];
+// const listitems = [
+//   {
+//     name: 'OnGoing',
+//     isOverdue: true,
+//     completed: false,
+//   },
+//   {
+//     name: 'OutDated',
+//     isOverdue: false,
+//     completed: true,
+//   },
+// ];
 
-  const [checked, setChecked] = useState([0]);
-  const [item, setItem] = useState('');
-  const [newItem, setNewItem] = useState(listitems);
+const JournalTodo = ({ data }) => {
+  const [todos, setTodos] = useState([]);
   const [anchorEl, setAnchorEl] = useState(null);
   const [cancel, setCancel] = useState([0]);
   // const [deleted, setDelete] = useState([0]);
```

Comment:
The code looks really clean overall but should the commented code be left in? If it's for future implementation then I would it's fine to leave in but otherwise I would be tempted to remove it.

Reply:
I was just focusing on the integration with backend. @AngelaLH and @cng822 are working on the refactor so I thought I would just leave this stuff to them.

Reply:
Ok sweet. In that case it looks good!